nvm.sh

```diff
@@ -61,18 +61,18 @@ nvm_command_info() {
   local COMMAND
   local INFO
   COMMAND="${1}"
-  if type "${COMMAND}" | nvm_grep -q hashed; then
-    INFO="$(type "${COMMAND}" | command sed -E 's/\(|\)//g' | command awk '{print $4}')"
-  elif type "${COMMAND}" | nvm_grep -q aliased; then
+  if nvm_has "${COMMAND}" | nvm_grep -q hashed; then
```

Comment:
```suggestion
  if command type "${COMMAND}" | nvm_grep -q hashed; then
```
One thing I noticed that if you change this to command type it returns the correct value and the test passes

Reply:
ahhh that actually makes some sense, since in this case, it's relying on the actual output of `type` (not just the exit code)

Reply:
Yeah I changed all of the instances of `nvm_has` to `command type` so I guess you might need to convert the remaining instances as well, but lets see if the tests pass :eyes:

Reply:
Yeah the tests fail : / so I guess you might have to replace all of the instances :P